### ISSUE TITLE
How to delete tag

### ISSUE THREAD

**Issue by medikoo** [photostructure/exiftool-vendored.js#104]
I wonder how to delete the existing tag. The documentation states just how to write (update or create tags), but there's no word on how specific tags can be removed

**Comment by mceachen** [comment]
Values can be removed by setting a value to `null`, like:

```ts
      await exiftool.write(src, { ExposureTime: null })
```

to erase `ExposureTime`, or 


```ts
      await exiftool.write(src, { AllTimes: null })
```

to erase all of the AllTimes"  tags (see the [documentation for this shortcut](https://sno.phy.queensu.ca/~phil/exiftool/TagNames/Shortcuts.html)

The typings don't currently suggest this is valid, though: the next release will include corrections to the `WriteTags` type. I hope to release in the next day or two with these typings changes and additional specs.

**Comment by medikoo** [comment]
@mceachen great thanks for explanations! It'll be nice to have that put somewhere in docs

**Comment by mceachen** [comment]
This is now released with version 15.7.0
